azure-pipelines/ui-automation/msal-broker-BrokerHost-nightly.yml

```diff
@@ -122,12 +102,12 @@ stages:
           - publish: $(Build.ArtifactStagingDirectory)/oldBrokerHost
             displayName: 'Publish Broker Host old apk for later use'
             artifact: oldBrokerHost
-  # MSAL with Broker Test Plan stage
-  - stage: 'msal_with_broker'
```

Comment:
this is hardcoded to API 30 but  it is based on **firebaseDeviceIdHigh** 
should we change it here and below?

(same with 29 and the other file)

Reply:
I'm a little confused what you mean. Where is API 30 is hardcoded? API 30+ is meant to imply any API level above 29. the idea is firebaseDeviceIdHigh and firebaseDeivceAndroidVersionHigh are meant to point to a firebase device with API level 30 or above, same thing with firebaseDeviceLowHigh but for API 29 and below

Reply:
OK I understood now. 
yeah, I was wondering if we should display which specific version it is running instead of API 30 +, or maybe both 
example something like this:
displayName: Running MSAL with Broker Test UI Test Suite (API 30+) {{parameters.firebaseDeviceIdHigh}}

Reply:
Ohhhh I see what you mean. That's a good suggestion! I think that it is good to include